app/instance-initializers/ember-intl.js

```diff
@@ -1,6 +1,6 @@
 export function initialize(appInstance) {
   const intl = appInstance.lookup('service:intl');
-  intl.setLocale('en-us');
+  intl.setLocale('fr-fr');
```

Comment:
Could you revert this change?

Reply:
Oopsie, commited a bit too fast ! Reverting !